===================================================================
RCS file: /cvs/gnome/nautilus/src/file-manager/fm-tree-view.c,v
retrieving revision 1.24
retrieving revision 1.25
diff -u -r1.24 -r1.25
--- fm-tree-view.c	2006/03/18 07:27:37	1.24
+++ src/file-manager/fm-tree-view.c	2006/04/25 12:25:20	1.25
@@ -1251,6 +1251,7 @@
 	
 	volume_monitor = gnome_vfs_get_volume_monitor ();
 	volumes = gnome_vfs_volume_monitor_get_mounted_volumes (volume_monitor);
+	volumes = g_list_sort (volumes, (GCompareFunc) gnome_vfs_volume_compare);
 	for (l = volumes; l != NULL; l = l->next) {
 		add_root_for_volume (view, l->data);
 		gnome_vfs_volume_unref (l->data);
